lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -500,10 +500,15 @@ def max_thread_limit
   end
 
   sha = `git rev-parse HEAD`[0..7]
+  branch = `git branch`
```

Comment:
Might be better to use `git rev-parse --abbrev-ref HEAD`

Doesn't require any magic text parsing.